src/inputs/actions.js

```diff
@@ -1,6 +1,7 @@
 import OutriggerAPI from '../utils/api';
 import _ from 'underscore';
 
+export const CLEAR_INPUTS = 'CLEAR_INPUTS';
```

Comment:
From a naming perspective, clear could be interpreted as clearing values. maybe `REMOVE_INPUTS` instead? Don't like either one _that_ much. Alternatively you could have the `clearInputs` action creator return

```
{
  type: INPUT_DEFS_UPDATE,
  payload: []
}
```